libpam/include/security/_pam_macros.h

```diff
@@ -94,6 +94,18 @@ do {                                              \
 #define _PAM_LOGFILE "/var/run/pam-debug.log"
 #endif
 
+#ifdef PAM_NO_HEADER_FUNCTIONS
+#ifdef UNUSED
+UNUSED
+#endif
```

Comment:
Given that `UNUSED` is always defined when Linux-PAM is built, let's use it unconditionally when `PAM_NO_HEADER_FUNCTIONS` is defined.

Reply:
Adjusted